public/docs/_examples/server-communication/dart/pubspec.yaml

```diff
@@ -5,12 +5,12 @@ version: 0.0.1
 environment:
   sdk: '>=1.19.0 <2.0.0'
 dependencies:
-  angular2: 2.0.0-beta.22
+  angular2: ^2.0.0
   browser: ^0.10.0
   dart_to_js_script_rewriter: ^1.0.1
-  http: ^0.11.3+3
```

Comment:
The ^ means it will pickup the latest 0.11.x ... so the rest after 0.11 is placebo :)